File: src/sdk-reference/php/3/kuzzle/query/snippets/query-1.php

<?php
use \Kuzzle\Kuzzle;

$kuzzle = new Kuzzle('localhost');

$queryArgs = [
  'controller' => 'controller',
  'action' => 'action'
];

$query = [
  'body' => [
    'foo' => 'bar'
  ],
  'other' => 'argument'
];

try {
  $response = $kuzzle->query($queryArgs, $query);
}
catch (ErrorException $e) {

}

File: src/sdk/android/3/core-classes/kuzzle/query/snippets/query-1.java

QueryArgs args = new QueryArgs();
args.controller = "controller";
args.action = "action";

JSONObject query = new JSONObject()
  .put("body", new JSONObject()
    .put("foo", "bar")
  )
  .put("other", "argument");

kuzzle.query(args, query, new OnQueryDoneListener() {
  @Override
  public void onSuccess(JSONObject object) {

  }

  @Override
  public void onError(JSONObject error) {
    // Handle error
  }
});
